deployer/src/deployer/upload.py

```diff
@@ -387,7 +388,9 @@ def on_task_complete(task):
                 if task.skipped:
                     totals.skipped += 1
                 elif task.error:
-                    failed_tasks.append(task)
+                    totals.failed += 1
+                    if show_progress_bar:
```

Comment:
What's done is done but I wonder if you ever want to or should use the progress bar. 
Even if was hacking on this on my laptop, I would probably prefer the no-progress where it just logs line after line. If I don't like I can pipe it to a file or something. 
In fact, for Yari, which also as a progress bar, I never use it because I've put `BUILD_NO_PROGRESSBAR=true` in my `.env`.

Reply:
I do find it nice for local testing full-blown uploads of tens of thousands of files where I don't care about the details (other than errors).